src/test/java/parser/JsonParserTest.java

```diff
@@ -1,53 +1,74 @@
 package parser;
-
-import org.junit.jupiter.api.*;
-import org.junit.jupiter.params.ParameterizedTest;
-import org.junit.jupiter.params.provider.ValueSource;
+import org.testng.annotations.*;
+import org.testng.asserts.SoftAssert;
 import shop.Cart;
 
 import java.io.File;
 import java.io.IOException;
 import java.nio.file.Files;
 import java.nio.file.Paths;
 
-class JsonParserTest {
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.assertThrows;
+
+public class JsonParserTest {
 
     private final static String EXPECTED_RESULT = "{\"cartName\":\"Test\",\"realItems\":[],\"virtualItems\":[],\"total\":0.0}";
     private JsonParser jsonParser;
     private Cart cart;
 
-    @BeforeEach
+    @BeforeTest (groups = {"checkinTests"})
     void initializeCart() {
         jsonParser = new JsonParser();
         cart = new Cart("Test");
     }
 
-    @ParameterizedTest()
-    @ValueSource(strings = {"andrew-cart", "pavel-cart", "alex-cart", "eugen-cart", "viktor-cart"})
-    void testNoSuchFileException(String name) {
-        Assertions.assertThrows(NoSuchFileException.class, () -> {
+    @DataProvider(name = "fileName")
+    public Object[][] dataProviderMethod() {
+        return new Object[][]{
+                {"andrew-cart"},
+                {"pavel-cart"},
+                {"alex-cart"},
+                {"eugen-cart"},
+                {"viktor-cart"}
+        };
+    }
+
+
+    @Test(dataProvider = "fileName")
+    public void testNoSuchFileException(String data) {
+        assertThrows(NoSuchFileException.class, () -> {
+            jsonParser.readFromFile(new File("src/main/resources/" + data + ".json"));
+        });
+    }
+
+    //@Ignore -> as discussed @Ignored can be used as well, but leaving 'enabled = false' as per request
```

Comment:
you can remove this commented code. `@Ignore` annotation is used in junit,` enabled = false` -- testng. Your current task is to use testng